portscout.conf

```diff
@@ -175,8 +175,10 @@ db connstr       = DBI:Pg:dbname=%(db_name)
 # GitHub site handler settings
 # GitHub rate limits requests to its API to a very low number for unauthenticated
 # requests, and 5000 per hour for authenticated requests.
+# GitHub personal access tokens can be requested on github accounts that
+# have a verified email address here: https://github.com/settings/tokens
+# A public personal access token without any special permissions will do(!)
 
-# github_client_id	=				# GitHub Client ID
-# github_client_secret	=				# GitHub Client Secret
+#github token     =				# GitHub personal access token
```

Comment:
Remove client_id/client_secret from code and replace it by github token (which really needs an underscore in here).